lib/cdo/crowdin/project.rb

```diff
@@ -0,0 +1,108 @@
+require 'httparty'
+
+module Crowdin
+  # This class represents a single project hosted on Crowdin, and provides
+  # access to data on that project via Crowdin's API
+  class Project
+    include HTTParty
+
+    attr_reader :id
+
+    # @param project_id [String]
+    # @param api_key [String]
+    # @see https://crowdin.com/project/codeorg/settings#api for an example of
+    #  how to retrieve these values for the "code.org" project
+    def initialize(project_id, api_key)
+      @id = project_id
+      self.class.base_uri("https://api.crowdin.com/api/project/#{project_id}")
+      self.class.default_params key: api_key
+    end
+
+    # @see https://support.crowdin.com/api/info/
+    def project_info
+      self.class.post("/info")
+    end
+
+    # @param file [String] name of file (within crowdin) to be downloaded
+    # @param language [String] crowdin language code
+    # @param etag [String, nil] the file version tag returned by crowdin; can
+    #  be used to check for updates without re-downloading the whole file every
+    #  time
```

Comment:
This comment explains what an etag is, but it wasn't clear to me on first read what it means to pass one as a parameter to `export_file`.  Maybe something like this?

> the last file version tag returned by crowdin for this file. If no changes have occurred since the provided etag was generated, crowdin will return a 304 (Not Modified) status instead of downloading the file.

Reply:
oh, good call! I'll also drop in a link to the relevant crowdin documentation